static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -0,0 +1,47 @@
+// @flow
```

Comment:
Oh yeah, I'm not opinionated on this. I called this a `FilterVisibilityToggle` because searchkit uses the name 'filter' for the various components for narrowing down the search (e.g. list menus, checkboxes, etc).